samples/client/petstore/typescript-rxjs/builds/default/runtime.ts

```diff
@@ -47,12 +47,18 @@ export class Configuration {
 
     get apiKey(): ((name: string) => string) | undefined {
         const apiKey = this.configuration.apiKey;
```

Comment:
It would be cool if you could already include these changes so avoid future merge conflicts: https://github.com/OpenAPITools/openapi-generator/pull/5465/files#diff-6230332f5c9f92f92f53b16d952c60b2R38-R39 and https://github.com/OpenAPITools/openapi-generator/pull/5465/files#diff-6230332f5c9f92f92f53b16d952c60b2R43-R44

Reply:
You bet.

Reply:
I don't think double ternary is appropriate there. It's obfuscating the conditional, it makes it less readable. But the logic is valid — I think webpack can take care of the compressing the code.